src/renderers/webvr/WebVRManager.js

```diff
@@ -107,10 +112,6 @@ function WebVRManager( renderer ) {
 
 			poseObject.position.fromArray( pose.position );
 
-		} else {
-
-			poseObject.position.set( 0, 0, 0 );
```

Comment:
Why did you remove this?

Reply:
I think it is redundant since I've never seen a null position because it's either emulated via head/neck model or coming from the positional tracking. If null is a case we want to cover we should do it here as well:
https://github.com/mrdoob/three.js/pull/13158/files#diff-fdee169d721b24ed863f98ee8b9f67f4R117

Reply:
I understand. Can we leave that for another PR?